src/ics/component.py

```diff
@@ -46,6 +46,10 @@ def to_container(self, context: Optional[ContextDict] = None) -> Container:
         return ComponentMeta.BY_TYPE[type(self)].serialize_toplevel(self, context)
 
     def serialize(self, context: Optional[ContextDict] = None) -> str:
+        """Will return a :class:`str` object
+        Component content as a serialized string fit for file write.
```

Comment:
From an aesthetical point, I'd suggest to insert a linebreak between the title ("will return...") and the description ("Component content...").

After a closer look, I think the first line is redundant. You have the return type already in the function signature. Why repeating it in the docstring? I would also shorten the docstring:

```suggestion
        """Creates a serialized string fit for file write.
```